teos/internal_api.py

```diff
@@ -137,10 +139,36 @@ def get_appointment(self, request, context):
 
     def get_all_appointments(self, request, context):
         with self.rw_lock.gen_rlock():
-            watcher_appointments = self.watcher.db_manager.load_watcher_appointments()
-            responder_trackers = self.watcher.db_manager.load_responder_trackers()
+            watcher_appointments = self.watcher.get_all_watcher_appointments()
+            responder_trackers = self.watcher.get_all_responder_trackers()
 
         appointments = Struct()
         appointments.update({"watcher_appointments": watcher_appointments, "responder_trackers": responder_trackers})
 
         return GetAllAppointmentsResponse(appointments=appointments)
+
+    def get_tower_info(self, request, context):
```

Comment:
Missing docs for all the methods. A one-liner would do (even if it points to the docs of another class)

Reply:
Added docs in 7d75df8. As the methods are basically service handlers, I only put one-liners on what it's doing, without details on the specific request format and response.